jumanji/environments/logic/rubiks_cube/env.py

```diff
@@ -12,29 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import List, Optional, Sequence, Tuple
+from typing import Optional, Sequence, Tuple
 
 import chex
 import jax
 import jax.numpy as jnp
-import matplotlib
 import matplotlib.animation
-import matplotlib.pyplot as plt
 
-import jumanji.environments
 from jumanji import specs
 from jumanji.env import Environment
-from jumanji.environments.logic.rubiks_cube.constants import (
-    DEFAULT_STICKER_COLORS,
-    CubeMovementAmount,
-    Face,
+from jumanji.environments.logic.rubiks_cube.constants import Face
+from jumanji.environments.logic.rubiks_cube.env_viewer import (
+    DefaultRubiksCubeViewer,
+    RubiksCubeViewer,
```

Comment:
I could but it breaks the point of being able to configure the env viewer

Reply:
Btw in case it's not clear, DefaultRubiksViewer is the one that is actually implemented whereas RubiksCubeViewer is the abstract class

Reply:
Apologies, my question should have been why you can't put `DefaultRubiksViewer` in the type annotation for now. This would only be temporary since I'll create a PR with a viewer interface to be used across all environments. But it's probably best to do that PR before merging this one. Let me know if you agree/disagree

Reply:
My only argument against this (aside from wanting to close this PR asap lol) is that some env viewers might be a bit less generic than the base class you have in mind - eg mine has a cube_size. It's not strictly necessary but I figured that any viewer for Rubik's cube would have this at a minimum.

Reply:
Wouldn't type hinting make it seem like you can pass another env's viewer when you really can't?

Reply:
@TristanKalloniatis it wouldn't - because each environment has a specific `State` - and this is specified in the type annotation `Viewer[State]`